news/6705.bugfix

```diff
@@ -0,0 +1 @@
+Fix inconsistent behavior of trusted host with port between HTTP and HTTPS indexes.
```

Comment:
I would be a bit more specific as to the concrete issue this is fixing / addressing. How about something like, "Fix ``--trusted-host`` processing under HTTPS to trust any port number used with the host." Would you say that's an accurate description?

Reply:
Sure, will do